conf/config.yaml

```diff
@@ -14,14 +14,15 @@ apisix:
   router:
     http: 'r3_uri'                # r3_uri: match route by uri(base on r3)
```

Comment:
the default router should be `radixtree`

Reply:
we can do this after this PR was merged few weeks.